bower.json

```diff
@@ -37,7 +37,8 @@
     "tos-pp": "https://github.com/mozilla/legal-docs.git",
     "underscore": "1.8.3",
     "webrtc-adapter": "0.2.5",
-    "vat": "0.0.5"
+    "vat": "0.0.5",
+    "Duration.js": "duration.js#~3.6.0"
```

Comment:
we need to make sure to lock this version of the dependency to avoid issues in the future, so just remove the `~`. (Just like other ones in this file)